admin/tests/integration/components/organizations/target-profiles-section_test.js

```diff
@@ -78,6 +79,89 @@ module('Integration | Component | organizations/target-profiles-section', functi
         .dom(screen.getByRole('link', { name: 'Number of The Beast' }))
         .hasAttribute('href', '/target-profiles/666');
     });
+
+    module('when detaching a target profile from an organization', function () {
+      test('it should display an Actions column to detach target profile', async function (assert) {
+        // given
+        const publicTargetProfileSummary = store.createRecord('target-profile-summary', {
+          id: 666,
+          name: 'Number of The Beast',
+        });
+        const privateTargetProfileSummary = store.createRecord('target-profile-summary', {
+          id: 777,
+          name: 'Super Lucky',
+        });
+        const organization = store.createRecord('organization', {
+          id: 1,
+          targetProfiles: [],
+          targetProfileSummaries: [publicTargetProfileSummary, privateTargetProfileSummary],
+        });
+
+        this.set('organization', organization);
+
+        // when
+        const screen = await render(hbs`<Organizations::TargetProfilesSection @organization={{this.organization}} />`);
+
+        //then
+        assert.ok(screen.getByText('Actions'));
+        const row = screen.getByRole('cell', { name: 'Super Lucky' }).closest('tr');
+        assert.ok(within(row).getByRole('button', { name: 'Détacher' }));
+      });
+
+      test('it should open confirm modal when click on "Détacher" button', async function (assert) {
+        // given
+        const targetProfileSummary = store.createRecord('target-profile-summary', {
+          id: 666,
+          name: 'Number of The Beast',
+        });
+        const organization = store.createRecord('organization', {
+          id: 1,
+          targetProfiles: [],
+          targetProfileSummaries: [targetProfileSummary],
+        });
+
+        this.set('organization', organization);
+        // when
+        const screen = await render(hbs`<Organizations::TargetProfilesSection @organization={{this.organization}} />`);
+        const detachButton = screen.getAllByRole('button', { name: 'Détacher' })[0];
+        await click(detachButton);
+        await screen.findByRole('dialog');
+        //then
+        assert.ok(screen.getByRole('heading', { name: "Détacher le profil cible de l'organisation" }));
+      });
+
+      test('it should detach a target profile when click on "Détacher" button', async function (assert) {
```

Comment:
**suggestion** : Renommer "it should detach a target profile when click on "Détacher" button" en "it should detach a target profile when click on "Confirmer" button"